### PR TITLE
Custom Gradle home and add .vpython file

### PR DESCRIPTION
**Pull request by blasten**
Custom the Gradle home and add `.vpython` script 

Issue: https://github.com/flutter/flutter/issues/55326.

### CODE REVIEW

File: testing/scenario_app/gradle-home/.vpython [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#
```
Comment (godofredoc) [inline testing/scenario_app/gradle-home/.vpython:1]: I think this needs to go in the location where the .py is but we can give it a try with led to see if this location works.

Comment (blasten) [reply testing/scenario_app/gradle-home/.vpython:1]: Based on https://chromium.googlesource.com/infra/infra/+/refs/heads/master/doc/users/vpython.md#common-specification:

> If an individual or embedded specification cannot be found, vpython will probe walk filesystem towards root (or .gclient root) looking for a common specification file. This file must be named .vpython and be located in or above the directory of the invoked script.

File: testing/scenario_app/android/gradle-home/bin/python [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+# As a result, vpython cannot find the .vpython spec file because it's outside
+# of the jar file.
+
+set -e
+
+SCRIPT_DIR="$( dirname "${BASH_SOURCE[0]}" )"
+
+echo "Running from $SCRIPT_DIR/bin/python"
+
+vpython -vpython-spec "$SCRIPT_DIR/../.vpython" "$@"
```
Comment (blasten) [inline testing/scenario_app/android/gradle-home/bin/python:14]: This was discussed with the LUCI infra folks and they said it's as better as it could get.

**Comment by blasten** [comment]
@godofredoc This PR is ready for a second look. 

You can see the recipe changes in https://flutter-review.googlesource.com/c/recipes/+/5520.  

Sample run: https://ci.chromium.org/raw/build/logs.chromium.org/flutter/led/egarciad_google.com/1ae22aaf69cd667f66f0769c4c988cf9d3dbdeee452331f2c91e7ba88dfb4ce1/+/annotations?server=chromium-swarm.appspot.com

File: testing/scenario_app/android/gradle-home/bin/python (hunk 2 above)
Comment (godofredoc) [reply testing/scenario_app/android/gradle-home/bin/python:14]: Great to see that it worked!

File: testing/scenario_app/android/gradle-home/.vpython [hunk 3]
```diff
@@ -0,0 +1,11 @@
+#
+# "vpython" specification file for build
+#
+# Pillow is required by com.facebook.testing.screenshot.
+
+python_version: "2.7"
```
Comment (dnfield) [inline testing/scenario_app/android/gradle-home/.vpython:6]: Any chance we can use 3, or does that need a separate PR?

Comment (blasten) [reply testing/scenario_app/android/gradle-home/.vpython:6]: The plugin that compares the screenshots requires `python-2.7`

**Pull request merged**